deploy/helm/values.yaml

```diff
@@ -437,3 +444,53 @@ aws_fargate:
     # see https://docs.fluentbit.io/manual/pipeline/filters
     # NOTE: The FluentBit configuration expects four spaces as indentation within sections
     filters:
+
+ebpfNetworkMonitoring:
+  enabled: false
```

Comment:
We should enable it by default.

Reply:
We need to discuss this offline, there are several risks involved in that